### PR TITLE
#3708 [Manager] Refactor and fix unable to display connector names in Toolbox on initial load

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3708)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Pipeline/Graph/Graph.js [hunk 1]
```diff
@@ -369,7 +369,7 @@ const Graph = props => {
                   classInfo,
                   isFetch: true,
                   isMetricsOn,
-                  metrics: targetCell ? targetCell.metrics : [],
+                  metrics: targetCell && targetCell.metrics,
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Graph/Graph.js]: A default parameter is used when `undefined` is supplied.

File: ohara-manager/client/src/context/api/workerApi.js [hunk 2]
```diff
@@ -145,7 +145,14 @@ export const createApi = context => {
         const res = await workerApi.start(params);
         if (res.errors) throw new Error(res.title);
 
-        const data = generateClusterResponse({ values: res.data });
+        const infoRes = await inspectApi.getWorkerInfo(params);
```
Comment (konekoya) [inline ohara-manager/client/src/context/api/workerApi.js]: Kudo to @saivirtue

File: ohara-manager/client/src/components/Node/AddNodeDialog.js [hunk 3]
```diff
@@ -80,7 +80,7 @@ const AddNodeDialog = props => {
                 validate={composeValidators(required, maxLength(63))}
               />
 
-              {mode !== MODE.k8s && (
+              {mode === MODE.docker && (
```
Comment (konekoya) [inline ohara-manager/client/src/components/Node/AddNodeDialog.js]: This reads better in my mind...

File: ohara-manager/client/src/context/api/workerApi.js (hunk 2 above)
Comment (saivirtue) [reply ohara-manager/client/src/context/api/workerApi.js]: Happy pair-programming 😆

**Pull request merged**